### DISCUSSION TITLE
Quire 1.0 deploy to GitHub Pages

### DISCUSSION THREAD

**Discussion by yl5682** [thegetty/quire#676] [Development]
I used to be using the `bin/deploy.sh` command to deploy a project to GitHub as GitHub Pages. However, I am unable to find this option in the 1.0.x version. I checked the v1 documentation but it seems that the deploy segment still refers to the same content as in v0. May I please get some advice from the community with regard to deployment?

Thank you.

**Comment by geealbers** [comment]
@yl5682, yes, that `bin/deploy.sh` script had gotten into v0 mostly because we were testing it out and just kind of left it in there. It was never a proper part of the project. I know it was useful though, so it might be worth considering adding something like that in v1, but I think it should likely be integrated into the CLI. (Could be a great community contribution!).

In the meantime you could try the following, just note that Quire sites deployed on GitHub Pages will suffer the known subdirectory path issue (#642) until we can push a release to fix that.

1. Download and unzip this file and add it to your project (a `bin` directory with a `deploy.sh` script inside): [bin.zip](https://github.com/thegetty/quire/files/10829961/bin.zip)
2. Change the `url:` in `content/_data/publication.yaml` to the GitHub Pages URL where your project will live
3. Run `bin/deploy.sh`

I _think_ it should work, but haven't tested it myself.
